src/script/components/Conversation/useFilesUploadDropzone/useFilesUploadDropzone.ts

```diff
@@ -60,7 +62,9 @@ export const useFilesUploadDropzone = ({
 
   const [accept, setAccept] = useState<Accept | undefined>(undefined);
 
-  const MAX_SIZE = isTeam ? CONFIG.MAXIMUM_ASSET_FILE_SIZE_TEAM : CONFIG.MAXIMUM_ASSET_FILE_SIZE_PERSONAL;
+  const TEAM_MAX_SIZE = isCellsEnabled ? CONFIG.MAXIMUM_ASSET_FILE_SIZE_CELLS : CONFIG.MAXIMUM_ASSET_FILE_SIZE_TEAM;
```

Comment:
Variable name is a little bit misleading.
```suggestion
  const MAX_ASSET_SIZE = isCellsEnabled ? CONFIG.MAXIMUM_ASSET_FILE_SIZE_CELLS : CONFIG.MAXIMUM_ASSET_FILE_SIZE_TEAM;
```

Reply:
Sorry, merged too soon :|
The next line makes the intent a bit clearer, there's 2 max size of assets for a team conversation : 500MB (cells) or 100MB. For a conv not owned by a team, it's always 15MB